### PR TITLE
Create ticket api integration

### PR DESCRIPTION
**Pull request by mudziok**
Bumped question length to 2000 as 255 seemed very little, I wonder if we need this sort of limit in the first place.

### CODE REVIEW

File: frontend/modules/shared/data-access/src/lib/services/ticket.service.ts [hunk 1]
```diff
@@ -42,4 +45,8 @@ export class TicketService extends AbstractApiService<TicketWithResponses> {
       headers: { 'Content-Type': 'application/json' },
     });
   }
+
+  createTicket(newTicket: CreateTicketDto) {
+    return this._http.post(`${this._resourceUrl}`, newTicket);
```
Comment (AdrianKokot) [inline frontend/modules/shared/data-access/src/lib/services/ticket.service.ts:50]: If there's no further logic, you can make `TikcetService` to extend `AbstractApiService<Ticket, CreateTicketDto>`. The 2nd generic parameter is the POST create request. Then the `create` method will do the same.

File: frontend/src/app/university/feature/page-contact-form/page-contact-form.component.ts [hunk 2]
```diff
@@ -47,24 +47,27 @@ import { first, switchMap } from 'rxjs';
...
-    name: ['', [Validators.required, Validators.maxLength(255)]],
-    subject: ['', [Validators.required, Validators.maxLength(255)]],
-    email: [
+    title: ['', [Validators.required, Validators.maxLength(255)]],
+    requesterEmail: [
       '',
       [Validators.required, Validators.maxLength(255), Validators.email],
     ],
-    content: ['', [Validators.required, Validators.maxLength(255)]],
+    description: ['', [Validators.required, Validators.maxLength(2000)]],
```
Comment (AdrianKokot) [inline frontend/src/app/university/feature/page-contact-form/page-contact-form.component.ts:59]: > Bumped question length to 2000 as 255 seemed very little, I wonder if we need this sort of limit in the first place.

I set this to 255 as the `Ticket.java` has a `String description` field, which default limit is 255 characters. `Response` content's also `String`. If you want to extend this limitation, I think you must adjust the model's properties to Postgres text column type.

**Review by AdrianKokot — CHANGES_REQUESTED** [review]
To check the `Ticket`/`Response` model's string properties

File: frontend/modules/shared/data-access/src/lib/services/ticket.service.ts (hunk 1 above)
Comment (mudziok) [reply frontend/modules/shared/data-access/src/lib/services/ticket.service.ts]: Did not know about that :D, changed

File: frontend/src/app/university/feature/page-contact-form/page-contact-form.component.ts (hunk 2 above)
Comment (mudziok) [reply frontend/src/app/university/feature/page-contact-form/page-contact-form.component.ts]: Sure I'll revert this change at least for the time being and address this sometime else

**Review by AdrianKokot — APPROVED** [review]

**Pull request merged**